src/injected/visualization/svg-drawer.ts

```diff
@@ -298,6 +298,7 @@ export class SVGDrawer extends BaseDrawer {
             centerPosition,
             drawerConfig.erroredTabIndexLabel,
             item.tabOrder,
+            item.itemType === TabbedItemType.MissingItem ? 'X' : null,
```

Comment:
I think it'd be cleaner if we set the circle configuration (line 291, just flip the condition) and the inner text under the same if block.